cc-cloudfuse

```diff
@@ -7,13 +7,24 @@ fi
 shift
 ARGS=$@
 
-# check auth url
-if ! [ -z ${OS_AUTH_URL:+x} ]; then
-    if [[ $OS_AUTH_URL != *?v3 ]]; then
-        OS_AUTH_URL="$OS_AUTH_URL/v3"
-    fi
+OPENSTACK_VENDOR_DATA='http://169.254.169.254/openstack/latest/vendor_data.json'
+OPENSTACK_VENDOR_DATA_2='http://169.254.169.254/openstack/latest/vendor_data2.json'
+
+JSON_VENDOR_DATA=$(curl -s $OPENSTACK_VENDOR_DATA_2)
+if [ "$JSON_VENDOR_DATA" = '{}' ]; then
```

Comment:
I didn't realize `=` could work, but in fact I learned that `==` is actually a Bash-specific thing, and the POSIX way is `=`. So, this is the best way! Cool.